README.md

```diff
@@ -2,87 +2,30 @@
 
 > **NOTE**: keelson is in the early phases of development and will undergo significant changes before reaching v1.0. Be aware!
 
-keelson is an opinionated key-space design and message format on top of [zenoh](https://github.com/eclipse-zenoh/zenoh).
+keelson is an opinionated key-space design, pub-sub message format and generic RPC interfaces on top of [zenoh](https://github.com/eclipse-zenoh/zenoh) primarily targeting the area of Maritime Robotics. It is provided as free and open-source software under the Apache 2.0 License.
```

Comment:
Marine Robotics is a Norwegian company :) 
```suggestion
keelson is an opinionated key-space design, pub-sub message format and generic RPC interfaces on top of [zenoh](https://github.com/eclipse-zenoh/zenoh) primarily targeting the area of Maritime vessel operation onboard and ashore. It is provided as free and open-source software under the Apache 2.0 License.
```

Reply:
Fair enough, will rephrase to avoid confusion.